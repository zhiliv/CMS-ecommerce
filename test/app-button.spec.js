import { shallowMount } from '@vue/test-utils'
import P from '@/components/app/button/button.vue'

describe('app-button.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(P, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        type: 'primary',
        size: 'lg'
      },
      slots:{
        default: 'Содержимое'
      }
    })
  })
  it('Содержимое html для app-button ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
