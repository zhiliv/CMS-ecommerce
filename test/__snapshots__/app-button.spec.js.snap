// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-button.js Содержимое html для app-button  1`] = `
<button
  class="btn btn-primary btn-lg simple simple-xl"
  id="indent"
  style="background: white;"
>
  Содержимое
</button>
`;
